ui/lib/core/addon/components/key-value-header.hbs

```diff
@@ -4,7 +4,7 @@
     {{#each this.secretPath as |path index|}}
       <li class={{if (is-active-route path.path path.model isExact=true) "is-active"}}>
         <span class="sep">&#x0002f;</span>
-        {{#if this.linkToPaths}}
+        {{#if (and this.linkToPaths (not this.isEngine))}}
```

Comment:
this is not used in engines as I have it currently setup. If I do use this conditional, then I'll need to modify the nested LinkTo to handle engine routing.

Reply:
Nope. Because the default is false, and PKI is the only engine passing the param as true.